x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/PkiRealmBootstrapCheckTests.java

```diff
@@ -121,4 +148,41 @@ public void testBootstrapCheckWithClosedSecuredSetting() throws Exception {
         secureSettings.close();
         assertThat(check.check(createTestContext(settings, null)).isFailure(), Matchers.equalTo(expectFail));
     }
+
+    public void testBootstrapCheckVerificationMode() throws Exception {
```

Comment:
assumeFalse for FIPS because `xpack.security.transport.ssl.verification_mode` is `none`